src/chess/app/chess.hoon

```diff
@@ -615,7 +620,22 @@
             ::  remove our challenge from challenges-sent
             challenges-sent  (~(del by challenges-sent) src.bowl)
             ::  put our new game into the map of games
-            games  (~(put by games) game-id.action [new-game *chess-position *(map @t @ud) | | | | | | src.bowl practice-game.u.challenge])
+            games  %-  %~  put  by
+                     games
+                   :-  game-id.action
+                   :*  new-game
+                       *chess-position
+                       *(map @t @ud)
+                       %.n
+                       %.n
+                       %.n
+                       %.n
+                       %.n
+                       %.n
+                       %.n
+                       src.bowl
+                       practice-game.u.challenge
+                   ==
```

Comment:
Must be a better way to do this than having a bunch of `%.n`s. Maybe bunt the various game attributes?